modules/tinymce/src/themes/silver/test/ts/browser/throbber/ThrobberEditorTest.ts

```diff
@@ -0,0 +1,72 @@
+import { UiFinder } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/mcagar';
+import { SugarBody } from '@ephox/sugar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.themes.silver.throbber.ThrobberEditorTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+  }, [ Theme ]);
+
+  const setProgressState = (editor: Editor, state: boolean, time?: number) => {
+    if (state) {
+      editor.setProgressState(true, time);
+    } else {
+      editor.setProgressState(false);
+    }
```

Comment:
```suggestion
    editor.setProgressState(state, time);
```
This is overly complicated and can be slimmed down to the above. Realistically you probably don't even need this function and could call `setProgressState` directly.

Reply:
Thanks. Have fixed this across all of the test files